browser/brave_wallet/BUILD.gn

```diff
@@ -20,5 +23,6 @@ source_set("brave_wallet") {
       "//chrome/browser/extensions:extensions",
       "//extensions/browser",
     ]
+    deps += brave_browser_wallet_deps
```

Comment:
this won't include `"//components/keyed_service/content:content"` if extensions are disabled